zulipterminal/core.py

```diff
@@ -254,8 +255,10 @@ def show_edit_history(
             'area:msg'
         )
 
-    def view_in_browser(self, message_id: int) -> None:
-        url = '{}#narrow/near/{}'.format(self.model.server_url, message_id)
+    def view_in_browser(self, message: Message) -> None:
+        # Truncate extra '/' from the server url.
+        url = near_message_url(self.model.server_url[:-1], message)
```

Comment:
Echoing Neil's comment from the last PR, how do you feel about `urljoin`?

Reply:
I am not too sure what the *pre-existing* code was there, but I am guessing it was something similar to our `server_url.py`?
We could use `urljoin`, but that will require completely refactoring our `server_url` code, and I am skeptical since the server uses the same approach as we have currently.

Reply:
Right. @neiljp Any thoughts on this?

Reply:
@zee-bit Could you point to the particular server code you're referring to?

Reply:
Re-reading this PR now I see that we've got `server_url.py` providing this functionality, which as @zee-bit says is from the server code :+1: